lessons/hash_to_array.rb

```diff
@@ -0,0 +1,20 @@
+my_hash = {}
+
+5.times do |i|
+    puts "Enter #{i + 1} key name:"
+    temp_key = gets.chomp
+    puts "Enter #{i + 1} value data:"
+    temp_value = gets.chomp
+    my_hash[temp_key] = temp_value
+end
+
+def hash_to_array(hash)
+    print "Keys array: " 
+    print hash.keys
+    puts
```

Comment:
I imagine you added this puts to create a new line.

You can clean up this code with either a newline character (`\n`) in your string or just as a single line with puts using string interpolation with `#{}`:
```suggestion
    puts "Keys array: #{hash.keys} " 
```

https://www.geeksforgeeks.org/how-to-add-new-line-in-ruby/